matter/efr32/efr32mg24/BRD2601B/autogen/sl_component_catalog.h

```diff
@@ -31,15 +31,19 @@
 #define SL_CATALOG_IOSTREAM_PRESENT
 #define SL_CATALOG_MEMORY_MANAGER_PRESENT
 #define SL_CATALOG_MEMORY_PROFILER_API_PRESENT
+#define SL_CATALOG_MPU_PRESENT
 #ifdef CONFIG_USE_EXTERNAL_FLASH
 #define SL_CATALOG_MX25_FLASH_SHUTDOWN_USART_PRESENT
 #endif // CONFIG_USE_EXTERNAL_FLASH
 #define SL_CATALOG_NVM3_PRESENT
+#define SL_CATALOG_OT_GP_INTERFACE_PRESENT
+#define SL_CATALOG_OPENTHREAD_STACK_PRESENT
 #define SL_CATALOG_POWER_MANAGER_PRESENT
 #define SL_CATALOG_PSA_CRYPTO_PRESENT
 #define SL_CATALOG_RAIL_LIB_PRESENT
 #define SL_CATALOG_RAIL_UTIL_PTI_PRESENT
 #define SL_CATALOG_SE_MANAGER_PRESENT
+#define SL_CATALOG_SECURITY_MANAGER_PRESENT
```

Comment:
This component significantly increases codesize. We want to avoid it as much as possible. Is this a hard req? (Going to hurt CMP pretty badly).

Reply:
Same here, it's a _could_.